src/routes/(infoPages)/[topTierPage]/page.scss

```diff
@@ -1,7 +1,3 @@
-.service-group-card .usa-icon {
-  filter: invert(1);
-}
```

Comment:
Not sure if using the same `page.scss` file is the right approach, but it works for now. 

We no longer need this rule since we now handle icon colors properly (albeit messily) in the `Button` component styling.